packages/react/.storybook/index.css

```diff
@@ -97,3 +97,41 @@ body {
     width: 100%;
   }
 }
+
+/* STEPPER */
+
+.stepper-custom-primary-button {
+  background-color: var(--color-black-90) !important;
```

Comment:
Should we use the theme variables rather than overwrite the CSS properties? It would show better example for the library users?

Reply:
Good suggestion, I had forgotten that black theme exists for button.

- [ ] Use black theme instead of overwriting the CSS properties